### PR TITLE
Bundle using `microbundle`

### PR DESCRIPTION
**Pull request by SirPryderi**
# Summary
- Replace commonjs `require` with `import/export`  statements
- Build bundles using `microbundle`
- Tweak tests to work with the new import configuration

Now the CDN bundled version of `wrld.js` is always minified, and the `wrld.min.js` version was dropped.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ example/
 node_modules/
 tests/
 tmp/
+covarage/
```
Comment (iainbethune) [inline .eslintignore:6]: typo here.

Comment (SirPryderi) [reply .eslintignore:6]: Whoops, thanks! 😬

**Review by iainbethune — APPROVED** [review]
Nice bit of modernisation.  I believe it all works correctly, but I picked up a couple of inconsistencies in how we define and export functions, so it's worth going back and checking over these.  Note: may be my lack of JS knowhow and what you've done is completely correct and for good reason.  Examples of each case below.

`heatmap_module.js` - `HeatmapModule` is a `function` and a default export
`routing_module.js` - `RoutingModule` is a `var` and a default export
`indoor_map_floor_outline_polygon.js` - `IndoorMapFloorOutlinePolygon` is `export const` and a default export
`elevation_mode.js` - functions are `export`ed but not `const`
`map_move_events.js` - `MapMoveEvents` is a `const` and default export

Not sure if it makes any difference in practice, but I suspect these can be made consistent and `export const` should be used when a function is defined, even if we default export it?

It's not like the original code was entirely consistent either, of course 🤣

**Comment by SirPryderi** [comment]
@iainbethune I can make the module definition consistent, but there are a few things to note:

1. Back when wrld.js was written, JavaScript had no class declaration. You could declare a _class_ by creating a function (which would act as a constructor) and defining functions within in as methods. This can only be achieved with the _function_ keyword and not the arrow function declaration (`const foo = () => {}`).
2. Using the `const foo = function () {}` (anonymous function) effectively delays the function definition (prevents hoisting), adding it after to the scope. We don't have any good reason to prefer this over just `function foo() {}` so I'll prefer the latter because it's shorter.
3. Now that we are using a modern bundling tool, we could safely use the class notation (VSCode automatically suggests to transpile the function class to a class definition), however I don't want to risk introducing new bugs at this stage.

In conclusion, I'll use the `function foo () {}` for defining modules and classes, but with the long term plan of converting them to proper class definitions when porting them to TypeScript.

**Review by iainbethune — APPROVED** [review]
